### ISSUE TITLE
Where is fetch_bounding_boxes_and_labels defined?

### ISSUE THREAD

**Issue by martinbel** [guigzzz/Keras-Yolo-v2#1]

**Comment by guigzzz** [comment]
It isn't defined anywhere, it is a placeholder. I explain the format of bounding_boxes and labels in the readme, so as long as your data follows that format, the code should work the same.

I also wanted to note that I haven't actually used this code to train a full model, so there is no guarantee that it will actually work (even though I was able to overfit on a simple example). 

However inference should work fine, so you should be able train your model with darknet, and then infer using this code

**Comment by martinbel** [comment]
It's a pity the function to read at least one common annotation format isn't implemented. You can probably adapt [this](https://github.com/experiencor/keras-yolo2/blob/master/preprocessing.py) to fit the project without much effort. 
Beside this, everything looks great! The code is very intuitive and simple to extend.

**Comment by guigzzz** [comment]
I would guess that tf's standard library has helper functions to parse common annotation formats, if not it's not very difficult to find ressources online. This repo was a learning experience for me, and not much more. I'm happy you like the code. If you find any bugs/mistakes don't hesitate to create a PR :)
